queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
This query will find animals with 'mon' in between their names.
- [ ] kindly update query to find animals with names that end with 'mon'

_`hint` only one `%` is needed_